src/dune_rules/coq/coq_path.ml

```diff
@@ -48,34 +48,15 @@ let config_path ~default coq_config key =
     (* This should never happen *)
     Code_error.raise "key is not a path" [ (key, Coq_config.Value.to_dyn path) ]
 
-let stdlib_plugins_dir path =
-  let open Memo.O in
-  let path = Path.relative path "plugins" in
-  let* dir_contents =
-    Fs_memo.dir_contents (Path.as_outside_build_dir_exn path)
-  in
-  match dir_contents with
-  | Error _ -> Memo.return []
-  | Ok dir_contents ->
-    let f (d, kind) =
-      match kind with
-      | File_kind.S_DIR | S_LNK -> Some (Path.relative path d)
-      | _ -> None
-    in
-    Memo.return
-      (List.filter_map ~f (Fs_cache.Dir_contents.to_list dir_contents))
-
-let build_user_contrib ~cmxs ~vo ~subpath ~name =
-  let path = subpath in
+let build_user_contrib ~cmxs ~vo ~path ~name =
   { name; path; cmxs; vo; stdlib = false }
 
 (* Scanning todos: blacklist? *)
 let scan_vo_cmxs ~path dir_contents =
   let f (d, kind) =
     match kind with
-    (* Skip some directories as Coq does, for now '-' and '.' *)
+    (* Skip some files as Coq does, for now files with '-' *)
     | _ when String.contains d '-' -> List.Skip
```

Comment:
Wasn't the code before skipping only directories containing a `.`?

Reply:
Before the PR the code was not skipping anything, I don't know what happened with older versions as git blame is not useful due to squashes.